deploy/docker/docker-entrypoint.sh

```diff
@@ -126,7 +136,7 @@ do
         # Config in plugins/*
         for CONFIG_PLUGINS_FILE in $(ls $CONFIG_PLUGINS); do
             if [[ ! -z "$(cat $CONFIG_PLUGINS/$CONFIG_PLUGINS_FILE |grep -E "^(^|^#*|^#*\s*)$VAR_NAME")" ]]; then
-                echo "$VAR_NAME=$(eval echo \$$VAR_FULL_NAME)"
+                echo "$(echo_value)"
```

Comment:
Same as 129 line

Reply:
resolved in 6bd5ed4f145e957bf35d3b9f567e1a3600224407